hammerblade/torch/pycosim.sh

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/bash -x
 
-COSIM_PYTHON_EXE=$BRG_BSG_BLADERUNNER_DIR/bsg_replicant/testbenches/pytorch/test_loader
+COSIM_PYTHON_EXE=$BRG_BSG_BLADERUNNER_DIR/bsg_replicant/examples/python/test_loader
```

Comment:
@vb000 I know v4.1.0 is not finalized, but my question is, are we going to use `python` instead of `pytorch`? I mean it makes sense as they are identical

Reply:
Did I remove something I shouldn't have?

Reply:
The folder is still there, but the Makefile is not working. I tried modifying it according to the one under python but I didn't get it to work. I'm sure I missed something there ...

Reply:
OK. So I probably skipped updating it since I didn't realize it was critical.

Reply:
Python is okay I think, because all we need is a way to start Python interpreter.